egui_demo_lib/src/easy_mark/easy_mark_editor.rs

```diff
@@ -117,59 +117,28 @@ impl EasyMarkEditor {
 
 fn shortcuts(ui: &Ui, code: &mut dyn TextBuffer, ccursor_range: &mut CCursorRange) -> bool {
     let mut any_change = false;
-    for event in &ui.input().events {
-        if let Event::Key {
+    for (key, surrounding) in [
+        (Key::B, "*"),
+        (Key::C, "`"),
+        (Key::I, "/"),
+        (Key::R, "~"),
+        (Key::U, "_"),
```

Comment:
You made a non-equivalent change, is this expected? Also I'd prefer to kept comments:
```suggestion
        (Key::B, "*"), // *bold*
        (Key::C, "`"), // `code`
        (Key::I, "/"), // /italics/
        (Key::L, "$"), // $lowered$
        (Key::R, "^"), // ^raised^
        (Key::S, "~"), // ~strikethrough~
        (Key::U, "_"), // _underline_
```

Reply:
Haha, I just realised and commited this at the same time